frontend/src/components/NotesDialog.js

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Button from "@material-ui/core/Button"
+import Dialog from "@material-ui/core/Dialog"
+import DialogActions from "@material-ui/core/DialogActions"
+import DialogContent from "@material-ui/core/DialogContent"
+import DialogContentText from "@material-ui/core/DialogContentText"
+import DialogTitle from "@material-ui/core/DialogTitle"
+import TextField from "@material-ui/core/TextField"
+
+function NotesDialog({ visibleDialog, toggleVisibleDialog }) {
+  const [participantNotes, setParticipantNotes] = React.useState("")
```

Comment:
we are using Mobx instead of state hooks...  but actually I think this is fine for now and we can later on add Mobx to this component.  The reason being, is the Mobx store that this will use is on a different branch that has yet to be PRed and I dont want to hold this up.

Reply:
hey @marcelkooi  - mind adding some tests and resolving this?

Reply:
Tests added

Reply:
@marcelkooi Tests look good!